docs/variants/novacustom_ns5x/releases.md

```diff
@@ -14,6 +14,42 @@ For details about our release process please read
 Test results for this platform can be found
 [here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit?usp=sharing).
 
+## v1.2.0 - 2022-05-26
+
+Test results for this release can be found
+[here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=1695997523).
+
+### Added
+
+- Persistent RGB keyboard settings
+- Fix for CVE-2022-29264
+- Fix BIOS vendor name in SMBIOS
```

Comment:
@mkopec If this says `fix`, it should be rather in a `fixed`  section? And be linked to an issue.

Reply:
Moved and added the link